internal/integration/base/k8s.go

```diff
@@ -236,13 +236,15 @@ type podInfo interface {
 	WithNodeName(nodeName string) podInfo
 	Create(ctx context.Context, waitTimeout time.Duration) error
 	Delete(ctx context.Context) error
+	SetQuiet(quiet bool)
```

Comment:
nit, to match previous `With*` methods, I'd make it:

```suggestion
	WithQuiet(quiet bool) podInfo
```

This allows method chaining